WL.VR/mission.sqm

```diff
@@ -359,7 +359,7 @@ class Mission
 					id=18;
 					side="WEST";
 					vehicle="B_Soldier_SL_F";
-					player="PLAY CDG";
+					player="PLAYER COMMANDER";
```

Comment:
This isn't how player is properly defined. You need to have it be "PLAY CDG" for a unit to be playable in a scenario. Go ahead and change this in your PR if you can, or we'll do it once it goes to dev, if it does.